### ISSUE TITLE
Lab 5 - confusing backpropagation explanation

### ISSUE THREAD

**Issue by Franckevicius** [jakelever/textasdata_bugbounty#45]
[Link to cell](https://colab.research.google.com/drive/1iMwqZENREnu1nkXYMXMWyd8o8h43Ks-X#scrollTo=pZdWopZAfjo2)

> 4. Use the calculated loss to get the gradients of the weights in the neural network (such that **moving along those gradients will reduce the loss**).
> 5. Move the weights slightly **along those gradients**.

Strictly speaking, we should be moving *in the opposite direction* of the gradient, not along them.

**Comment by jakelever** [comment]
Agreed. This was awkwardly worded. Have reworded. 3 points
